pkg/update/testdata/setters/expected/kustomization.yaml

```diff
@@ -2,8 +2,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - unimportant.yaml
```

Comment:
With the new version of kyaml the indentation behavior changed. Are we ready for this?

Reply:
Let's pin kyaml to the old version please.